code/game/objects/items/handcuffs.dm

```diff
@@ -197,6 +197,22 @@
 /obj/item/restraints/handcuffs/cable/zipties/used/attack()
 	return
 
+/obj/item/restraints/handcuffs/cable/zipties/beepsky
+	name = "beepsky zipties"
+	desc = "Cheap plastic disposable zipties that beepsky use to handcuff criminals."
+	icon_state = "cuff"
+	custom_materials = null
+	breakouttime = 150
+	trashtype = /obj/item/restraints/handcuffs/cable/zipties/beepsky/used
+
+/obj/item/restraints/handcuffs/cable/zipties/beepsky/used
+	desc = "A pair of broken zipties."
+	icon_state = "cuff_used"
+	item_state = "cuff"
+
+/obj/item/restraints/handcuffs/cable/zipties/beepsky/used/attack()
+	return
```

Comment:
whats the point of this if its exactly the same as the other type of zipties anyway?

Reply:
Makes sense. I deleted it.